example/angular/styles/angular-example.css

```
@@ -0,0 +1,143 @@
+@import url('../../styles/reset.css');
```

Stylelint (17.3.0)

[error] Expected "url('../../styles/reset.css')" to be "'../../styles/reset.css'" (import-notation)

(import-notation)